ironfish/src/rpc/clients/tcpClient.test.ts

```diff
@@ -23,6 +23,7 @@ describe('IronfishTcpClient', () => {
         data: {
           mid: messageId,
           type: route,
+          auth: 'test token',
```

Comment:
I don't think we should pass this in any old test, to ensure they work as is. Especially since it should not be required.